.github/workflows/ci.yml

```
@@ -0,0 +1,42 @@
+name: Contract Development
+
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+      - 0.[0-9]+
+
+jobs:
+  test-hackatom:
+    name: ${{ matrix.build }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        include:
+          - build: macOS
+            os: macOS-latest
+          - build: Windows
+            os: windows-latest
+    defaults:
+      run:
+        shell: bash
+        working-directory: ./contracts/hackatom
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+      - name: Install Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: 1.47.0
```

What do you think about
```suggestion
          toolchain: stable
```
?

---

Two things:
1. I prefer having deterministic CI builds, i.e. we get the same results when running them in two months from now
2. It is unlikely that a new Rust version in the 1.x range breaks source codes that worked before. However, it is easy to write code that only works with the latest stable, but not older versions. For this reason I'd always test the lowest supported Rust version.

---

I see, thanks for the clarification.